src/components/shapesAndOntologies/ShapesAndOntologies.tsx

```diff
@@ -0,0 +1,101 @@
+import styles from "./ShapesAndOntologies.module.css";
+import Title from "../Title/Title";
+import { useTranslation } from "react-i18next";
+import { useContext, useEffect, useState } from "react";
+import { AuthContext } from "../../context/AuthContextProvider";
+import car from "../../assets/car.gif";
+import { ApiService } from "../../services/ApiService";
+import { mapOntologies, mapShapesAndOntologies, Ontology } from "../../utils/dataMapper";
+import SelfDescriptionCard from "../cards/SelfDescriptionCard";
+import { AuthContextValues } from "../../context/AuthContextValues";
+import Text from "../Text/Text";
+import SearchBar from "../searchBar/SearchBar";
```

Comment:
Let's reorganize the imports here as well to maintain a similarity between components.